src/main/java/projet/projetstage02/service/GestionnaireService.java

```diff
@@ -409,4 +408,139 @@ public ContractsDTO getContractsToEvaluateMillieuStage() {
+        evaluationParagraph.put("L'environnement de travail respecte les normes de sécurité",
+                fieldToText(evaluationMillieuStage.getEnvironementTravail()));
+        evaluationParagraph.put("Le climat de travail est agréable",
+                fieldToText(evaluationMillieuStage.getClimatTravail()));
+        evaluationParagraph.put("Le millieu de stage est accessible par transport en commun",
+                fieldToText(evaluationMillieuStage.getMilieuDeStage()));
+        evaluationParagraph.put("La communication avec le superviseur est efficace",
+                fieldToText(evaluationMillieuStage.getCommunicationAvecSuperviser()));
+        evaluationParagraph.put("L'équipement de travail est adéquat",
+                fieldToText(evaluationMillieuStage.getEquipementFourni()));
+        evaluationParagraph.put("Le volume de travail est raisonnable",
+                fieldToText(evaluationMillieuStage.getVolumeDeTravail()));
+
+        commentaires.put("", evaluationMillieuStage.getCommentaires());
+
+        signPara.put("Signé le", evaluationMillieuStage.getDateSignature());
+        map.put("Information sur la compagnie", companyInfo);
+        map.put("Information sur l'étudiant", studentInfo);
+        map.put("Information sur l'offre de stage", offerInfo);
+        map.put("Évaluation", evaluationParagraph);
+        if (evaluationMillieuStage.getCommentaires() != null && !evaluationMillieuStage.getCommentaires().isEmpty()) {
+            map.put("Commentaires", commentaires);
+        }
+        map.put("Signature", signPara);
+        map.put("_signature_", getSignatureGestionnaire(evaluationMillieuStage));
+        return map;
+    }
+
+    private Map<String, String> getSignatureGestionnaire(EvaluationMillieuStage evaluationMillieuStage) throws EmptySignatureException {
+        String signature = evaluationMillieuStage.getSignature();
+        String[] split = signature.split(",");
+        if (split.length != 2) {
+            throw new EmptySignatureException();
+        }
+        signature = split[1];
+        if (signature.length() < 2) throw new EmptySignatureException();
+        Base64.Decoder decoder = Base64.getDecoder();
+        byte[] decodedByteArray = decoder.decode(signature);
+        String bytes = byteToString(decodedByteArray);
+        Map<String, String> map = new LinkedHashMap<>();
+        map.put("signature", bytes);
+        return map;
+    }
+
+    public PdfOutDTO getEvaluationMillieuStagePDF(long id) throws NonExistentEntityException {
+        Optional<EvaluationPDF> optional = evaluationMillieuStagePDFRepository.findById(id);
+        if (optional.isEmpty()) throw new NonExistentEntityException();
+        EvaluationPDF evaluationPDF = optional.get();
+        return new PdfOutDTO(evaluationPDF.getContractId(), evaluationPDF.getPdf());
+    }
+
+    public List<StageContractOutDTO> getEvaluatedContractsMillieuStage() {
```

Comment:
Maybe rename this one getEvaluationMillieuStage because the one before is getEvaluationMillieuStagePDF